### ISSUE TITLE
New Repository

### ISSUE THREAD

**Issue by velikanov** [yankodimitrov/SwiftPasscodeLock#29]
@ziogaschr, @waltermvp, @venj, @kiokumicu, @guidelfrate, @gblotter, @karlml, @twofish187

Look guys, I just don't know what's happened with @yankodimitrov and I hope that everything is just going fine and he simply became very-very rich and threw out his laptop

But anyway, this great project needs an evolution, so I propose to continue work on this project somewhere else

I've made a fork of this project and merged a @ziogaschr outstanding commits into it, so you can fork it and start to make things happen

Thank you

**Comment by waltermvp** [comment]
@velikanov I share your sentiment.

**Comment by velikanov** [comment]
@waltermvp, excuse me, I thought that GitHub clones all repository structure automatically
it's done now, you can create new Issues
I'll try to migrate all old open issues

**Comment by ziogaschr** [comment]
@velikanov I agree, let's keep this project live.
We are actively using this library in an app and we are really happy.
I have add, some more functionality but in separate classes in my project.
I really find handy, that @yankodimitrov decided in the refactor to keep the PasscodeRepository as a protocol and not make it part of the project.

We have also added exponential backoff on failed PIN attempts + a dummyView for app is in background. ;)

Let me know if you want me to share some tips on this.

**Comment by ziogaschr** [comment]
@yankodimitrov if you like the idea you can add us as contributors so as we can help you and we keep this nice repo, instead of another fork

**Comment by ziogaschr** [comment]
@yankodimitrov I recently saw that you have accepted a PR. Will we keep this repo active? Shall we continue making our PR's to your repo?

This is your work man and you have done very good work so far, so please update us if we can help you to keep this project rolling.

**Comment by waltermvp** [comment]
+:1

**Comment by yankodimitrov** [comment]
First thank you all for your pull requests and support!
You can continue to send pull requests. Im a bit busy right now but will try to look at the repo.

Furthermore @ziogaschr and @velikanov let me know if you want me to add you as contributors.

Thanks!

**Comment by ziogaschr** [comment]
Thank you for the update @yankodimitrov.

It's up to you to add us as collaborators. :)

Do you find a good idea to merge the latest changes from @velikanov master? As it is harder to re-create all PRs. Let us know if you prefer PRs

**Comment by ziogaschr** [comment]
@yankodimitrov I tried to add new PR's although, because the base of what I currently use is from @velikanov master branch the PR had more stuff in it.

Is it ok with you to create a PR with all the changed things at @velikanov master branch so as we sync them and then add two new PRs https://github.com/velikanov/SwiftPasscodeLock/pull/15 and https://github.com/velikanov/SwiftPasscodeLock/pull/16 so as we can continue with your repo?

**Comment by waltermvp** [comment]
@ziogaschr  need to use this repo again. I'm assuming your fork is the best option to move forward? Also would using "PasscodeLock" for cocoapods give me your fork? if not how would i add your fork using cocoaopods?

**Comment by ziogaschr** [comment]
Yes. I have updated the Swift3 branch on my repo this week, but I want to check some stuff till monday in order I merge it with my repos' master.

I would like to help closing all the issues, although every time I am trying to do this I have to merge all changes from main repo + @velikanov which is such a pain

Sent from my iPhone

On 18 Nov 2016, at 21:09, Walter Martin Vargas-Pena <notifications@github.com<mailto:notifications@github.com>> wrote:

@ziogaschrhttps://github.com/ziogaschr need to use this repo again. I'm assuming your fork is the best option to move foward?

##

**Comment by waltermvp** [comment]
Hmm I can imagine. Here to help :)
